apps/test/unit/code-studio/progressTest.js

```diff
@@ -39,54 +35,3 @@ describe('bestResultLevelId', function() {
     assert.strictEqual(bestResultLevelId([4, 5], progressData), 5);
   });
 });
-
-describe('initViewAs', function() {
```

Comment:
This function was moved into `scripts/show.js` as a private function

Reply:
Should we add equivalent test coverage elsewhere or not necessary?

Reply:
The logic was moved to `show.js`, I realized we don't generally have unit tests for show files which is why I didn't add any tests. I think feature tests will capture whether we display the correct view for the user. Do you think I should introduce a test for `show.js`?

Reply:
I don't have a strong opinion.  Maybe have a quick chat with Dani since most of these tests cover functionality that she worked on recently?

Reply:
I think it would be good to try to keep some testing if we can. Especially since this is the code that makes sure `viewAs=teacher` still works and correctly points you to `viewAs=instructor` I would not want us to somehow lose that redirect accidentally because it could break a bunch of links